### PR TITLE
feat: synchronize community token between TokenMasters and Owners

### PR DESCRIPTION
**Pull request by mprakhov**
Synchronize community tokens between other TokenMasters and Owners.
TokenMasters or Owners require full CommunityToken data in their databases to manipulate token
Owners will not synchronize OwnerToken and TokenMasterTokens with other TokenMasters

This MR depends on:  https://github.com/status-im/status-go/pull/3879

Closes # https://github.com/status-im/status-desktop/issues/11715

### CODE REVIEW

**Comment by jrainville** [comment]
~~I think this PR closes https://github.com/status-im/status-desktop/issues/11610 ?~~

edit: nvm, this PR is about tokens. My bad

**Review by jrainville — APPROVED** [review]
Nice job

**Review by osmaczko — COMMENTED** [review]
Late into the party, looks ok :+1:

File: protocol/communities/token/community_token.go [hunk 1]
```diff
@@ -39,3 +42,54 @@ type CommunityToken struct {
...
+		ChainId:            int32(token.ChainID),
+		DeployState:        protobuf.CommunityToken_DeployState(token.DeployState),
+		Base64Image:        token.Base64Image,
+		Decimals:           int32(token.Decimals),
+		Deployer:           token.Deployer,
+		PrivilegesLevel:    protobuf.CommunityToken_PrivilegesLevel(token.PrivilegesLevel),
+	}
+}
+
+func FromCommunityTokenProtobuf(pToken *protobuf.CommunityToken) *CommunityToken {
```
Comment (endulab) [inline protocol/communities/token/community_token.go:67]: @mprakhov can't we use only protobuf.CommunityToken ?

**Pull request merged**